pkg/api/types.go

```diff
@@ -803,9 +803,10 @@ type Container struct {
 
 	// Variables for interactive containers, these have very specialized use-cases (e.g. debugging)
 	// and shouldn't be used for general purpose containers.
-	Stdin     bool `json:"stdin,omitempty"`
-	StdinOnce bool `json:"stdinOnce,omitempty"`
-	TTY       bool `json:"tty,omitempty"`
+	Stdin          bool `json:"stdin,omitempty"`
+	StdinOnce      bool `json:"stdinOnce,omitempty"`
+	TTY            bool `json:"tty,omitempty"`
+	ReadOnlyRootFs bool `json:"readOnlyRootFs,omitempty"`
```

Comment:
I'm not convinced that `readOnlyRootFS` is clearer that `readOnly` - any reason not to use the shorter name?